### PR TITLE
Continue fixing golint errors [T708]

### PR DESCRIPTION
**Pull request by vixentael**
Fixes include:
- added package headers here and there.
- added many comments to public functions/ types.
- fixed "if block ends with a return statement, so drop this else and outdent its block".
- changed "n += 1" to "n++".
- fixed naming for some fields (serverId -> serverID).

Errors number decreased from 788 to 512.

### CODE REVIEW

File: cmd/acra-authmanager/acra_authmanager.go [hunk 1]
```diff
@@ -83,7 +91,7 @@ func (hp HashedPasswords) SetPassword(name, password string) (err error) {
 	return nil
 }
 
-func ParseHtpasswdFile(file string, keystore *filesystem.FilesystemKeyStore) (passwords HashedPasswords, err error) {
+func parseHtpasswdFile(file string, keystore *filesystem.FilesystemKeyStore) (passwords HashedPasswords, err error) {
```
Comment (Lagovas) [inline cmd/acra-authmanager/acra_authmanager.go]: change to private because used only in this module?

File: decryptor/base/callbacks_test.go [hunk 2]
```diff
@@ -8,7 +8,7 @@ import (
 type TestCallback struct{ CallCount *int }
 
 func (callback *TestCallback) Call() error {
-	*callback.CallCount += 1
```
Comment (Lagovas) [inline decryptor/base/callbacks_test.go]: python style vs golang style))

File: cmd/acra-authmanager/acra_authmanager.go (hunk 1 above)
Comment (vixentael) [reply cmd/acra-authmanager/acra_authmanager.go]: yes

**Comment by vixentael** [comment]
> check and update variable name of main.TestOnly in tests/test.py

that's might be the reason of failing tests. thank you

**Comment by Lagovas** [comment]
awesome results) well done :)

**Pull request merged**